homeassistant/components/airnow/const.py

```diff
@@ -14,10 +14,32 @@
 ATTR_API_REPORT_DATE = "DateObserved"
 ATTR_API_REPORT_HOUR = "HourObserved"
 ATTR_API_REPORT_TZ = "LocalTimeZone"
-ATTR_API_REPORT_TZINFO = "LocalTimeZoneInfo"
 ATTR_API_STATE = "StateCode"
 ATTR_API_STATION = "ReportingArea"
 ATTR_API_STATION_LATITUDE = "Latitude"
 ATTR_API_STATION_LONGITUDE = "Longitude"
 DEFAULT_NAME = "AirNow"
 DOMAIN = "airnow"
+
+SECONDS_PER_HOUR = 3600
+
+# AirNow seems to only use standard time zones,
+# but we include daylight savings for completeness/futureproofing.
+US_TZ_OFFSETS = {
```

Comment:
Any chance AirNow will/can be used outside of the US?

Reply:
Airnow is a US government run service, so all the timezones it returns are the three-letter US specific ones. There are other (much better, imo) services for international data (like waqi). But I used AirNow for a while and noticed this issue, so wanted to fix it for correctness sake.